app-modules/assistant/resources/views/filament/pages/ai-assistant.blade.php

```diff
@@ -43,64 +47,64 @@
 
             <form wire:submit.prevent="sendMessage">
                 <label
-                    class="sr-only"
-                    for="chat"
+                        class="sr-only"
+                        for="chat"
                 >Your message</label>
                 <div class="flex items-center rounded-lg bg-gray-50 px-3 py-2 dark:bg-gray-700">
                     @if (!$chat->id)
                         <button
-                            class="inline-flex cursor-pointer justify-center rounded-lg p-2 text-gray-500 hover:bg-gray-100 hover:text-gray-900 dark:text-gray-400 dark:hover:bg-gray-600 dark:hover:text-white"
-                            type="button"
-                            wire:loading.attr="disabled"
-                            wire:click="save"
+                                class="inline-flex cursor-pointer justify-center rounded-lg p-2 text-gray-500 hover:bg-gray-100 hover:text-gray-900 dark:text-gray-400 dark:hover:bg-gray-600 dark:hover:text-white"
+                                type="button"
+                                wire:loading.attr="disabled"
+                                wire:click="save"
```

Comment:
Is this correct?

Reply:
Do you mean the formatting, or something else?

Reply:
These changes indent the attributes an extra level, I didn't think this is how HTML was formatted in this app